src/docs/samples/primer/kotlin/settings.gradle.kts

```diff
@@ -1 +1,26 @@
-pluginManagement.includeBuild("gradle/plugins")
+pluginManagement {
+    includeBuild("gradle/plugins")
+}
+
+/*
```

Comment:
I added this _commented out_ so that we can also have the multi-language sample here already. We can remove this once we upgraded the samples to `1.2` when released.

Reply:
Good idea 🙂